reana_workflow_engine_yadage/externalbackend.py

```diff
@@ -128,10 +92,10 @@ def submit(self, spec, parameters, state, metadata):
         if kerberos:
             job_request_body['kerberos'] = kerberos
 
-        job_id = self.rjc_api_client.submit(**job_request_body)
+        jobproxy = self.rjc_api_client.submit(**job_request_body)
```

Comment:
Why do we call this `jobproxy`? What we get from this call is the [clear response from RJC](https://github.com/reanahub/reana-commons/blob/a6e516a899bd64de994bfdf5e2981fce4009059a/reana_commons/api_client.py#L114) which [is the created `job_id`](https://reana-job-controller.readthedocs.io/en/latest/_static/api.html#operation/get_jobs).

Reply:
NVM, I just saw that we are passing the clear response so it would be `{"job_id": "cdcf48b1-c2f3-4693-8230-b066e088c6ac"}` assigned to jobproxy, and then in `packtivity` this key, `job_id`, is widely used for example [here](https://github.com/yadage/packtivity/blob/ac7f544a9f0e0e9cdafa7986c30e38e4ea4275bc/packtivity/asyncbackends.py#L123).